Makefile

```
@@ -47,18 +47,18 @@ css:
 
 %.pdf:
 	mkdir -p docs/data-model
-	postgresql_autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
+	postgresql_autodoc -l /usr/share/postgresql-autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
```

```suggestion
	postgresql_autodoc -l /usr/share/postgresql-autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
```
Why using ``-l /usr/share/postgresql-autodoc`` ?

---

no need to specify on my instance as it's default shared data for postgresql-autodoc

---

Ok, on ubuntu 22.04 ?

I installed autodoc especially for this purpose and by default it seems that the program was trying to reach another folder (as mentioned in this issue https://github.com/cbbrowne/autodoc/issues/26).
If this is only a bug on my computer I guess we should remove this commit then.